lib/app/modules/more/bindings/more_binding.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:get/get.dart';
+
+import '../controllers/more_controller.dart';
+
+class MoreBinding extends Bindings {
```

Comment:
use appropriate and meaningful page/module name.

Reply:
done